Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex

```diff
@@ -137,15 +141,26 @@ \section{Persistent data management}
 Moodcat has to handle both song data and chatroom data.
 
 \par
-For the song we store the basic properties, like artist, song title, genre and duration.
-Furthermore each song has its own \gls{valence} and \gls{arousal} value, which is computed and corrected by the music matching algorithm.
-The algorithm uses some of the audio features of the track for the computation of the valence and arousal values, these features are stored with the song.
+We try do deduplicate artists from the Soundcloud data as much as possible. We identify artists in our database by an unique identifier. Besides this identifier, we store the name for the artist.
```

Comment:
'Additionally we store the name of the artist in order to display on the frontend.'